### ISSUE TITLE
Entering shortcuts on mac inputs wrong characters

### ISSUE THREAD

**Issue by Ghoelian** [zen-browser/desktop#3698]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

When changing keyboard shortcuts on MacOS, the input shows characters that aren't the one I typed.

![image](https://github.com/user-attachments/assets/96287174-dd24-4ff5-9ee1-da2260d4a3cd)

Here, I entered the same keybinds as were the default (cmd+opt+U, H, V, and G).

The keybinds do still work, and this way the cmd+opt+U shortcut even works, not opening the website source as reported in #1738.

When I press the same keys in a normal text input field, those characters don't show up.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.1

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by Ghoelian** [comment]
Looks like these are the characters normally typed by pressing opt+key.

I'm not very familiar with macs, had no idea it did that.

**Comment by salamijason** [comment]
Strange, will send to devs.

**Comment by salamijason** [comment]
Are you sure you're clicking on the cmd key instead of the ctrl key?

**Comment by Eveeifyeve** [comment]
Yes I can confirm this is happening to me, the only fix is switching to unicode keyboard. @mr-cheff Maybe for macos could you switch it to `CMD+shortcuts` or `CMD+SHIFT+Shortcut`

**Comment by mr-cheffy** [comment]
Is this purely a visual bug then?

**Comment by Ghoelian** [comment]
@mr-cheff not entirely, I think. Cmd+opt+U does behave slightly differently after I "changed" it than it did before, not opening the page source afterwards.

**Comment by Ghoelian** [comment]
@salamijason yes, I'm sure I'm pressing Cmd+opt+keys.

**Comment by mr-cheffy** [comment]
Last question, what language is your keyboard setup as?

**Comment by Ghoelian** [comment]
It's set up as English U.S.

**Comment by stoicAlchemist** [comment]
I'm coming from https://github.com/zen-browser/desktop/issues/1738, looking at this, I think the issue is the input management, it does make sense if Opt+u inputs the ¨ char, I'm not sure if shortcut inputs are handled differently and CMD should be detected as priority modifier to not take Opt+u as ¨ but as a chain of modifiers or how that works in code actually. Let me know if you guys need to run tests and I'll try to provide as much info as I can.

**Comment by Eveeifyeve** [comment]
> I'm coming from #1738, looking at this, I think the issue is the input management, it does make sense if Opt+u inputs the ¨ char, I'm not sure if shortcut inputs are handled differently and CMD should be detected as priority modifier to not take Opt+u as ¨ but as a chain of modifiers or how that works in code actually. Let me know if you guys need to run tests and I'll try to provide as much info as I can.

I guess this is a Firefox issue, as Firefox needs to implement something like option as meta key like how terminals work. So I would say this is a firefox issue.

**Comment by Eveeifyeve** [comment]
> I'm coming from #1738, looking at this, I think the issue is the input management, it does make sense if Opt+u inputs the ¨ char, I'm not sure if shortcut inputs are handled differently and CMD should be detected as priority modifier to not take Opt+u as ¨ but as a chain of modifiers or how that works in code actually. Let me know if you guys need to run tests and I'll try to provide as much info as I can.

I guess this is a Firefox issue, as Firefox needs to implement something like option as meta key like how terminals work. So I would say this is a firefox issue.

So a fix is switch to unicode keyboard which is found in `inputs -> other -> unicode`, but I think this is most likely a glitch on firefox end and not zen browsers so I would report it to them.

also I would wait for @HarryHeres to verify if this is the case.

**Comment by Eveeifyeve** [comment]
I would wait for @HarryHeres to verify if this is the case.
